### PR TITLE
Literally two functions

### PR DESCRIPTION
**Pull request by adrienntindall**
this was hell

### CODE REVIEW

File: src/battle/overlay_12_0224E4FC.c [hunk 1]
```diff
@@ -3438,3 +3438,429 @@ int DamageDivide(int num, int denom) {
...
+    
+    do {
+        switch (ctx->sendOutState) {
+        case 0: //field weather 
+            if (!ctx->weatherCheckFlag) {
+                switch (ov12_0223BAF8(bsys)) {
+                case 1:
+                case 2:
+                case 3:
+                    script = 271;
```
Comment (red031000) [inline src/battle/overlay_12_0224E4FC.c:3461]: should these be consts? unsure if we have a naix for it already or not

File: src/battle/overlay_12_0224E4FC.c [hunk 2]
```diff
@@ -3438,3 +3438,429 @@ int DamageDivide(int num, int denom) {
...
+                    for (battlerIdCheck = 0; battlerIdCheck < maxBattlers; battlerIdCheck++) {
+                        if (BattleSystem_GetFieldSide(bsys, battlerId) != BattleSystem_GetFieldSide(bsys, battlerIdCheck) && !(ctx->battleMons[battlerIdCheck].status2 & STATUS2_SUBSTITUTE) && ctx->battleMons[battlerIdCheck].hp) {
+                            def += ctx->battleMons[battlerIdCheck].def * sStatChangeTable[ctx->battleMons[battlerIdCheck].statChanges[2]][0] / sStatChangeTable[ctx->battleMons[battlerIdCheck].statChanges[2]][1];
+                            spdef += ctx->battleMons[battlerIdCheck].spDef * sStatChangeTable[ctx->battleMons[battlerIdCheck].statChanges[5]][0] / sStatChangeTable[ctx->battleMons[battlerIdCheck].statChanges[5]][1];
+                        }
+                    }
+                    ctx->battleMons[battlerId].downloadFlag = TRUE;
+                    if (def + spdef != 0) {
+                        if (def >= spdef) {
+                            ctx->statChangeParam = 0x12;
```
Comment (red031000) [inline src/battle/overlay_12_0224E4FC.c:3597]: any idea what these params mean

File: src/battle/overlay_12_0224E4FC.c [hunk 3]
```diff
@@ -3438,3 +3438,429 @@ int DamageDivide(int num, int denom) {
...
+                        } else {
+                            ctx->statChangeParam = 0xF;
+                        }
+                        ctx->statChangeType = 3;
```
Comment (red031000) [inline src/battle/overlay_12_0224E4FC.c:3601]: same as above for these types

File: src/battle/overlay_12_0224E4FC.c [hunk 4]
```diff
@@ -3438,3 +3438,429 @@ int DamageDivide(int num, int denom) {
...
+                    for (battlerIdCheck = 0; battlerIdCheck < maxBattlers; battlerIdCheck++) {
+                        if (BattleSystem_GetFieldSide(bsys, battlerId) != BattleSystem_GetFieldSide(bsys, battlerIdCheck) && ctx->battleMons[battlerIdCheck].hp) {
+                            hp += ctx->battleMons[battlerIdCheck].hp;
+                            for (index = 0; index < LEARNED_MOVES_MAX; index++) {
+                                moveNo = ctx->battleMons[battlerIdCheck].moves[index];
+                                power = ctx->unk_334.moveData[moveNo].power;
+                                switch (power) {
+                                case 1:
+                                    switch (ctx->unk_334.moveData[moveNo].effect) {
+                                    case 38: //OHKO?
```
Comment (red031000) [inline src/battle/overlay_12_0224E4FC.c:3676]: constants?

**Review by red031000 — CHANGES_REQUESTED** [review]

File: src/battle/overlay_12_0224E4FC.c (hunk 1 above)
Comment (adrienntindall) [reply src/battle/overlay_12_0224E4FC.c:3461]: I tried to look for one but couldn't find it. Lhea seems to know what the consts are but I'm unsure if pret has ported those yet

File: src/battle/overlay_12_0224E4FC.c (hunk 2 above)
Comment (adrienntindall) [reply src/battle/overlay_12_0224E4FC.c:3597]: nope

File: src/battle/overlay_12_0224E4FC.c (hunk 4 above)
Comment (adrienntindall) [reply src/battle/overlay_12_0224E4FC.c:3676]: I'm only sure about the OHKO one, gonna wait till I can get more info about move effects before documenting these

File: src/battle/overlay_12_0224E4FC.c (hunk 3 above)
Comment (adrienntindall) [reply src/battle/overlay_12_0224E4FC.c:3601]: I think the types are like "Special attack up" but not sure

**Review by red031000 — APPROVED** [review]

**Pull request merged**